specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/examples/Events_ListBySingleResource.json

```diff
@@ -0,0 +1,104 @@
+{
+  "parameters": {
+    "resourceUri": "subscriptions/4abcdefgh-ijkl-mnop-qrstuvwxyz/resourceGroups/rhctestenv/providers/Microsoft.Compute/virtualMachines/rhctestenvV1PI",
+    "api-version": "2021-01-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.ResourceHealth/events/BC_1-FXZ",
```

Comment:
The APIs to get events are at subscription or resource level.  Is this ID appropriate?  If a user does a GET on this resource ID will they see the same content they see here?  If not, the ID should change to be GETable by the user

Reply:
same as above.

Reply:
Presumably its just the examples that are incorrect and are an easy fix.  If that is not the case there are deeper issues here that need to be addressed in your service

Reply:
Our service does not support querying of individual events based on the event id . we are using this id just as an identifier . The reason being the systems which we query for this information also do not support querying based on a single event id , The only supported operation is getting all events for a resource , Could you please let us know if we need to support querying by specific Id ?